README.md

```diff
@@ -66,6 +93,17 @@ On HPC cluster OBR can submit operations via the job queue. For example
 
 will submit the `blockMesh` operation to the cluster manager for every job that is eligible. OBR detects the installed job queuing system, eg. slurm, pbs, etc. A jobs ubmission script will be generated automatically. For fine grained control over the submission script the `--template` argument allows to specify the location of a submission script template. Since OBR uses signac for job submission more details on how to write job submission templates can be found [here](https://docs.signac.io/en/latest/templates.html). To avoid submitting numereous jobs individually, the `--bundling-key` argument can be used to bundle all jobs for which the bundling key has the same value into the same job.
 
+### 4. Postprocessing cases
+
+Since OBR aims at performing parameter studies containing a larger number of individual casses, postprocessing cases manually should be avoided. Its is recommended to use `obr apply` instead.
+
+    obr apply --file script.py --campaign ogl_170
+
+The passed `script.py` file must implement a `call(jobs: list[Job], kwargs={})` function. On execution this gets a list of jobs which allow access to the case paths.
+
+
```

Comment:
```suggestion
The mandatory `--campaign` argument is used to separate (families of) experiments. The optional argument `--tag <str>` can be used to further differentiate between experiments within a campaign. Internally, this will create subfolders as such:

    fd52708db5c296d1fa52b056701be4ee
        ├── campaign1
        │   ├── tag1
        │   │   ├── decomposePar_2024-01-05_17:38:44.log
        │   │   ├── instrumentedPimpleFoam_2024-01-05_17:44:32.log
        │   │   └── solverExitCode.log
        │   └── tag2
        │       ├── decomposePar_2024-01-05_17:38:44.log
        │       ├── instrumentedPimpleFoam_2024-01-05_17:44:32.log
        │       └── solverExitCode.log
        └── campaign2    
```